revolv/static/dashboard.scss

```diff
@@ -72,3 +91,116 @@
     max-width: 100%;
     float: left;
 }
+.project-container {
+    padding-left: 0px;
+    padding-right: 0px;
+    padding-top: 0px;
+    padding-bottom: 0px;
+    margin: 0px, 0px, 0px, 0px;
+
+    .status-bar {
+    transition: border-color 200ms linear;
```

Comment:
I also think there's a mixin for transition to alias it to `-webkit-transition` etc also for compatibility?